src/workflow_components/composite_action.py

```diff
@@ -27,6 +28,42 @@ def get_or_create_composite_action(path: str) -> "CompositeAction":
     return obj
 
 
+class CompositeActionInput(GraphObject):
+    __primarykey__ = "_id"
+
+    _id = Property()
+    name = Property()
+    default = Property()
+    description = Property()
+    required = Property()
+    url = Property()
+    action = RelatedTo("CompositeAction")
+
+    def __init__(self, _id: str, name: str, url: str):
```

Comment:
init should be like this

    def __init__(self, param: str, path: str):
        self.param = param
        self.path = path
        self._id = md5(f"{param}_{path}".encode()).hexdigest()

Reply:
The md5 calc is in `from_dict` of `CompositeAction`. The `_id` is already in md5 at init point